.gitmodules

```diff
@@ -4,3 +4,7 @@
 [submodule "g2html"]
 	path = g2html
 	url = https://github.com/goblint/g2html.git
+[submodule "gobview"]
+	path = gobview
+	url = https://github.com/goblint/gobview.git
+	branch = goblint
```

Comment:
I think this can be merged now, but maybe we should move the contents of `goblint` branch into Gobview's `master` branch and omit this alternative branch name here before we merge?